spyder/plugins/mainmenu/plugin.py

```diff
@@ -70,7 +70,8 @@ def on_initialize(self):
         create_app_menu(ApplicationMenus.Run, _("&Run"), dynamic=False)
         create_app_menu(ApplicationMenus.Debug, _("&Debug"), dynamic=False)
         create_app_menu(ApplicationMenus.Consoles, _("C&onsoles"))
-        create_app_menu(ApplicationMenus.Projects, _("&Projects"))
+        if self.get_conf("enable", section=Plugins.Projects):
```

Comment:
```suggestion
        if self.get_plugin(Plugins.Projects, error=False):
```

This is how we detect if a plugin is active or not.

Reply:
Using `get_plugin()` fails with `SpyderAPIError: Plugin "project_explorer" not part of REQUIRED or OPTIONAL requirements!`. I have opted for using `is_plugin_enabled()` which works, and fits even better.

Reply:
> Using get_plugin() fails with SpyderAPIError: Plugin "project_explorer" not part of REQUIRED or OPTIONAL requirements!

You're right, I forgot that in order to use `get_plugin` you need to declare it as a dependency.

> I have opted for using is_plugin_enabled() which works, and fits even better

Ok, that's a great idea! Thanks a lot for looking into it.